pkg/machine/machine.go

```diff
@@ -0,0 +1,69 @@
+package machine
+
+import (
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/containers/common/pkg/config"
+	"github.com/sirupsen/logrus"
+)
+
+type MachineMarker struct {
+	Enabled bool
+	Type    string
+}
+
+const (
+	markerFile = "/etc/containers/podman-machine"
+	Wsl        = "wsl"
```

Comment:
Can add a qemu type as well. I think we should write that file for qemu as well to not end up in and undefined state.